### ISSUE TITLE
Re-opening a closed PR causes pr-bumper to not find the PR sha

### ISSUE THREAD

**Issue by sglanzer** [ciena-blueplanet/pr-bumper#22]
Sometimes when you start a PR the Travis build won't start automatically.  A quick fix for this is to close the PR and reopen the same PR, which starts the travis build.  It must change the sha for the PR though, because pr-bumper was unable to find the PR.

Wish I had the log for this - will come back and paste it in the next time it happens.

**Comment by rajcoss** [comment]
I caught the log when I ran into this one...

```
$ pr-bumper check
No .pr-bumper.json found, using defaults
Using CI commit sha [a204837fcea5ebf249f3e4c68c2f22e5682d7675]
RO_GH_TOKEN = [4820766e94dfb85417cc4465dc4ba9c07282fc2c]
No PR found for sha [a204837fcea5ebf249f3e4c68c2f22e5682d7675]
The command "pr-bumper check" failed and exited with 1 during .
```

It would be useful to suggest the workaround (adding a commit) when such an error happens.

**Comment by job13er** [comment]
@sandersky fixed this with #24
